schema/site.schema.json

```
@@ -66,6 +66,12 @@
           "type": "string",
           "enum": ["enabled", "disabled"],
           "default": "disabled"
+        },
+        "interactiveSearchMode": {
```

```suggestion
        "splitSearchModes": {
```

Suggesting this change to avoid the following, which I find confusing:

```typescript
    showInteractiveSearchMode: boolean
    interactiveSearchMode: boolean
```